docs/getting-started.md

```diff
@@ -47,135 +51,164 @@ Radicle development is public by default and completely open-source. See
   </em>
 </details>
 
-### Supported Platforms
+## Install the Radicle CLI
 
-Radicle Upstream is available for Linux and macOS (version 10.14 and
-newer).
+The Radicle CLI installs on Linux, macOS, Windows-based systems by compiling from source or with Homebrew.
```

Comment:
Does it really work on Windows? Has anyone tested that?

Reply:
I just asked on Discord—we'll see if I get a response.